R/PIConfiguration.R

```diff
@@ -1,69 +1,73 @@
 #' @export
 #' @format NULL
 PIConfiguration <- R6::R6Class(
   "PIConfiguration",
   inherit = ospsuite.utils::Printable,
   cloneable = TRUE,
   active = list(
-    #' @field simulateSteadyState Boolean representing whether the simulation
-    #' should be brought to a steady-state first
+    #' @field simulateSteadyState Boolean for simulation steady-state first
     simulateSteadyState = function(value) {
       if (missing(value)) {
         private$.simulateSteadyState
       } else {
-        validateIsLogical(value)
+        ospsuite.utils::validateIsLogical(value)
         private$.simulateSteadyState <- value
       }
     },
-    #' @field steadyStateTime Time in minutes to simulate if simulating steady-state. May be NULL
+    #' @field steadyStateTime Time in minutes for steady-state simulation. May be NULL
     steadyStateTime = function(value) {
       if (missing(value)) {
         private$.steadyStateTime
       } else {
-        validateIsNumeric(value)
+        ospsuite.utils::validateIsNumeric(value)
         if (value < 0) {
           stop(paste0("steadyStateTime must be a positive numerical value, but the value is ", value))
         }
         private$.steadyStateTime <- value
       }
     },
-    #' @field printEvaluationFeedback Boolean. If `TRUE`, the objective function value
-    #' will be printed after each function evaluation.
-    #' Default is `FALSE`
+    #' @field printEvaluationFeedback Boolean. If `TRUE`, prints objective
+    #' function value after each evaluation. Default is `FALSE`
     printEvaluationFeedback = function(value) {
       if (missing(value)) {
         private$.printEvaluationFeedback
       } else {
-        validateIsLogical(value)
+        ospsuite.utils::validateIsLogical(value)
         private$.printEvaluationFeedback <- value
       }
     },
 
-    #' @field simulationRunOptions Object of type `SimulationRunOptions` that will be passed
-    #' to simulation runs. If `NULL`, default options are used.
+    #' @field simulationRunOptions Object of `SimulationRunOptions` for simulation
+    #' runs. If `NULL`, default options are used.
     simulationRunOptions = function(value) {
       if (missing(value)) {
         private$.simulationRunOptions
       } else {
-        validateIsOfType(value, "SimulationRunOptions", nullAllowed = TRUE)
+        ospsuite.utils::validateIsOfType(
+          value, "SimulationRunOptions", nullAllowed = TRUE
+        )
         private$.simulationRunOptions <- value
       }
     },
 
-    #' @field targetFunctionType Type of the target function used for error
-    #' calculation. Supported target functions  are listed  in
-    #' `ospsuite.parameteridentification::ObjectiveFunctions`.
-    targetFunctionType = function(value) {
-      if (missing(value)) {
-        private$.targetFunctionType
+    #' @field objectiveFunctionOptions Configuring model fit assessment
+    #' within `.objectiveFunction`. Influences error and model fit calculation.
```

Comment:
This documentation is visible to the user, while `.objectiveFunction` is a hidden (private) method. We should not mention any private fields in the user documentation. Just re-phrase it.

Reply:
I see your point. Fixed.